detection_rules/schemas/__init__.py

```diff
@@ -199,7 +205,7 @@ def get_stack_schemas(stack_version: str) -> Dict[str, dict]:
     versions = {k: v for k, v in stack_map.items()
                 if (mapped_version := Version(k)) >= stack_version and mapped_version <= current_package and v}
 
-    if stack_version > current_package:
+    if stack_version > current_package or not versions:
```

Comment:
this seems like it would mask important errors
```suggestion
    if stack_version > current_package:
```

Reply:
Good point. I restored it and replaced it with a test b53a9bf and a 7.15 entry with a `TODO`